tests/gtest/avif_fuzztest_dec_incr.cc

```diff
@@ -62,10 +62,15 @@ void DecodeIncr(const std::string& arbitrary_bytes, bool is_persistent,
   ASSERT_NE(decoder.get(), nullptr);
   avifDecoderSetIO(decoder.get(), &io);
   // OSS-Fuzz limits the allocated memory to 2560 MB.
+  constexpr uint32_t kMaxMem = 2560u * 1024 * 1024;
+  // Consider at most four planes of 16-bit samples.
+  constexpr uint32_t kMaxSampleMem =
```

Comment:
Nit/optional: This constant is the maximum image size in samples. It would be good for the constant's name to reflect that. Maybe `kMaxImageSize`?